e2e/fields/fields.ts

```diff
@@ -92,6 +94,16 @@ export class Fields {
 
         let container = this.container;
 
+        if (fieldLabel) {
+            await this.anyPage.contentContains(fieldLabel)
+        } else {
+            await this.anyPage.waitUntilLoaded();
+        }
+
+        if (complexFieldLabel) {
+            await this.anyPage.contentContains(complexFieldLabel)
+        }
```

Comment:
Would this benefit from an `else` like the other similar ones?

Reply:
I left the `else` out from here because this finder is used by pretty much every field type, so anything that wasn't a complex field type would suddenly have an extra delay of a few seconds, which would blow out the time to run in AAT.
And it seems to work without needing that additional waiting, so decided not to include at this point.